setup.py

```diff
@@ -77,7 +77,7 @@ def convert_to_py2():
 
 
 setup(name='PyLaTeX',
-      version='1.0.0',
+      version='1.0.3',
```

Comment:
Could you undo this change, I'll update the version myself when I do the next release.